docs/reference/api/lapacklike.rst

```diff
@@ -1,26 +1,24 @@
 .. meta::
-  :description: hipSOLVER documentation and API reference library
-  :keywords: hipSOLVER, rocSOLVER, ROCm, API, documentation
+  :description: hipSOLVER LAPACK-like functions API documentation
+  :keywords: hipSOLVER, rocSOLVER, ROCm, API, documentation, lapack-like
 
 .. _lapacklike:
 
 ********************************
 hipSOLVER LAPACK-like functions
 ********************************
 
-Other Lapack-like routines provided by hipSOLVER are divided into the following subcategories:
-
-* :ref:`likeeigens`. Eigenproblems for symmetric matrices.
-* :ref:`likesvds`. Singular values and related problems for general matrices.
-
+The other LAPACK-like routines provided by hipSOLVER are divided into the following subcategories:
```

Comment:
It would be nice if this description was consistent with dense-api/lapacklike.rst